src/helperFunctions.js

```diff
@@ -600,13 +600,15 @@ class helpers {
       method: 'GET',
       service: this.OCS_SERVICE_CLOUD,
       action: 'user',
-      data: null
+      data: null,
+      headers: []
     }
     options = Object.assign({}, defaults, options)
     const action = options.action.includes('?') ? options.action + '&format=json' : options.action + '?format=json'
     const url = this.instance + this.OCS_BASEPATH_V2 + options.service + '/' + action
     const headers = this.buildHeaders()
     headers['OCS-APIREQUEST'] = 'true'
+    headers.concat(options.headers)
```

Comment:
headers = headers.concat(options.headers)

<img width="97" alt="image" src="https://user-images.githubusercontent.com/277525/90859677-eed1b680-e388-11ea-9cc5-39a320f3e113.png">

Reply:
wait, I mixed it up with regular arrays

hash arrays don't have the `concat` option, did you test it ?

you could use `Object.assign()` instead like above

Reply:
yeah .... ci was already crying ..... bloody js